test-network-function/networking/suite.go

```diff
@@ -190,14 +193,18 @@ func runNetworkingTests(netsUnderTest map[string]netTestContext, count int) {
 				netUnderTest.testerSource.ip, aDestIP.containerIdentifier.PodName,
 				aDestIP.containerIdentifier.ContainerName,
 				aDestIP.ip))
-			testPing(netUnderTest.testerContainerNodeOc,
-				netUnderTest.testerSource.containerIdentifier.NodeName,
-				netUnderTest.testerSource.containerIdentifier.ContainerUID,
-				netUnderTest.testerSource.containerIdentifier.ContainerRuntime,
-				aDestIP.ip,
-				count)
+			testPass := testPing(netUnderTest.testerContainerNodeOc, netUnderTest.testerSource.containerIdentifier, aDestIP, count)
+			if !testPass {
+				if failedDestIps, netFound := badNets[netName]; netFound {
+					badNets[netName] = append(failedDestIps, aDestIP.ip)
+				} else {
+					badNets[netName] = []string{aDestIP.ip}
```

Comment:
If we don't have dest pod, we should probably include the source pod/ip at least?

Reply:
I already modified the claim file log trace to include both pod names and the target IP. See lines 284+288.